exocom-server/src/exocom.ls

```diff
@@ -34,6 +37,7 @@ class ExoCom extends EventEmitter
   # deregisters a service instance that went offline
   deregister-client: (client-name) ~>
     @client-registry.deregister-client client-name
+    @websocket.deregister-client client-name
```

Comment:
the kernel now contains the handler methods for all types of events. They orchestrate the different subsystems to handle the respective event.